streamz/sources.py

```diff
@@ -17,6 +17,7 @@ def _():
 
     pc = tornado.ioloop.PeriodicCallback(_, callback_time, **kwargs)
     pc.start()
+    pc.start()
```

Comment:
Why is this called the second time?

Also, as far as I can see, PeriodicCallback is only used in a single test (testing PeriodicCallback I think). Might as well remove it altogether.

Reply:
You would replace it with from_iter/buffer if you wanted this kind of source?

Reply:
Ah, it's a source. I thought it was some utility for something that was removed. I didn't even recognize it as such without `@Stream.register_api(staticmethod)`. Then I think that it's not necessarily redundant, but I'd expect to see it mounted as `Stream` method, like other sources, e.g. `Stream.from_periodic_callback`.

Reply:
I think it might be useful, as the simplest of sources along with from_iter. Its style could be made to fit the rest of the module:

```python
@Stream.register_api(staticmethod)
class from_periodic(Source):
    
    def __init__(self, callback, poll_interval=0.1, **kwargs)
        self.cb = callback
        self.poll = poll_interval
        super().__init__(**kwargs)

    async def _run(self):
        await asyncio.gather(self._emit(self.cb()))
        await asyncio.sleep(self.poll)
```

It is sink_to_file that I would remove, since we now have to_textfile in sinks.

Reply:
I agree, looks good.

As a side note, I think we need some clobbering protection for nodes (ref. #318). For example, if at some point we're going to have a node called `poll` in core, this will break. I'm personally using `_` at the beginning for all stream attrs, but it's really easy to forget/miss.